source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -128,10 +128,14 @@ def get_queryset(self):
 
     def get(self, request, *args, **kwargs):
         if self.request.user.has_perm('apiVolontaria.get_user'):
-            return self.retrieve(request, *args, **kwargs)
+            content = {
+                'detail': _("You are not authorized to get detail of a given user."),
+            }
+            return Response(content, status=status.HTTP_403_FORBIDDEN)
+            #return self.retrieve(request, *args, **kwargs)
```

Comment:
Why do you change this logic ? I don't think we want to change this endpoint, and if we want we need to discuss it and create an other PR

Reply:
oh shit, cetait pour faire des tests je l'avais comme il était avant je ne sais pas ce qui s'est passé. je change ca